tests/test_method_query.py

```diff
@@ -129,3 +167,21 @@ def _calc_output_dim_testmethod(non_slice_dims_shape, testparam):
         "httomo.methods_database.packages.external.sample.supporting_funcs.module.path"
     )
     assert dims == (10, 20)
+
+
+def test_database_query_calculate_padding(mocker: MockerFixture):
+    class FakeModule:
+        def _calc_padding_testmethod(size):
+            assert size == 5
+            return 5, 5
+
+    importmock = mocker.patch("importlib.import_module", return_value=FakeModule)
+    query = MethodsDatabaseQuery("sample.module.path", "testmethod")
+
+    pads = query.calculate_padding(size=5)
+
+    importmock.assert_called_once_with(
+        "httomo.methods_database.packages.external.sample.supporting_funcs.module.path"
+    )
+
+    assert pads == (5, 5)
```

Comment:
Nitpick, take it or leave it: whenever I see a repeated constant I prefer to make one variable to reference everywhere, makes it less likely to get stuff wrong if needing to edit the test later.

Could define `LEFT_PADDING` and `RIGHT_PADDING` via something like `LEFT_PADDING = RIGHT_PADDING = 5`, and then use them where appropriate, would make the test slightly more understandable than seeing magic 5's in places?

Maybe also defining `SIZE`, so then the `assert size == 5` and the `pads = query.calculate_padding(size=5)` are more obviously connected to some `SIZE` variable?